lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -627,6 +627,7 @@ class MagmadUtil(object):
     config_update_cmds = Enum("config_update_cmds", "MODIFY RESTORE")
     apn_correction_cmds = Enum("apn_correction_cmds", "DISABLE ENABLE")
     health_service_cmds = Enum("health_service_cmds", "DISABLE ENABLE")
+    enable_disable_options = Enum("enable_disable_options", "DISABLE ENABLE")
```

Comment:
I would prefer having separate Enums for both HA and IPv6 configurations, let's separate this and call each one `ipv6_config_cmds` and `ha_service_cmds`

Reply:
Done